inst/extdata/packages/CoordinateCleaner.yaml

```diff
@@ -0,0 +1,32 @@
+- name : CoordinateCleaner
+- title: Automated Cleaning of Occurrence Records from Biological Collections
+- version: 2.0-20
+- author: Alexander Zizka, Daniele Silvestro, Tobias Andermann, Josue Azevedo, Camila Duarte Ritter, Daniel Edler, Harith Farooq, Andrei Herdean, Maria Ariza, Ruud Scharn, Sten Svanteson, Niklas Wengstrom, Vera Zizka, Alexandre Antonelli
+- maintainer: Alexander Zizka
+- cran: yes
+- github: https://github.com/ropensci/CoordinateCleaner
+- description: Automated flagging of common spatial and temporal errors in biological and paleontological collection data, for the use in conservation, ecology and paleontology. Includes automated tests to easily flag (and exclude) records assigned to country or province centroid, the open ocean, the headquarters of the Global Biodiversity Information Facility, urban areas or the location of biodiversity institutions (museums, zoos, botanical gardens, universities). Furthermore identifies per species outlier coordinates, zero coordinates, identical latitude/longitude and invalid coordinates. Also implements an algorithm to identify data sets with a significant proportion of rounded coordinates. Especially suited for large data sets. The reference for the methodology is: Zizka et al. (2019) <doi:10.1111/2041-210X.13152>.
```

Comment:
The `:` are wrong in the yaml because they are used as a field delimiter. Please escape theim with `\`in "methodology is:".